etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.meta.yml

```diff
@@ -773,6 +772,8 @@ tables:
         unit: '% of population'
         display:
           name: Access to clean fuels and technologies for cooking
+        presentation:
+          title_public: Access to clean fuels and technologies for cooking (% of population)
```

Comment:
I'd say "Share of people with access to clean fuels and technologies for cooking".